### PR TITLE
Add "Ablaza Pawnshop"

### PR DESCRIPTION
**Pull request by govvin**
Add brand for Philippine-based chain of pawnbrokers, "Ablaza Pawnshop"

### CODE REVIEW

File: data/brands/shop/pawnbroker.json [hunk 1]
```diff
@@ -10,6 +10,15 @@
   },
   "items": [
     {
+      "displayName": "Ablaza Pawnshop",
+      "locationSet": {"include": ["ph"]}
```
Comment (tommylung) [inline data/brands/shop/pawnbroker.json:14]:
```suggestion
      "locationSet": {"include": ["ph"]},
```
Missing comma

File: data/brands/shop/pawnbroker.json [hunk 2]
```diff
@@ -10,6 +10,15 @@
...
+      "tags": {
+        "brand": "Ablaza Pawnshop",
+        "brand:wikidata": "",
```
Comment (tommylung) [inline data/brands/shop/pawnbroker.json:17]: We need to add the Wikidata for the brand.

**Review by tommylung — CHANGES_REQUESTED** [review]

File: data/brands/shop/pawnbroker.json (hunk 1 above)
Comment (Abrarhaj) [reply data/brands/shop/pawnbroker.json:14]: Ok

File: data/brands/shop/pawnbroker.json (hunk 2 above)
Comment (Abrarhaj) [reply data/brands/shop/pawnbroker.json:17]: Ok

Comment (govvin) [reply data/brands/shop/pawnbroker.json]: Oopsie. That's an oversight on my part. Thanks @tommylung

**Pull request merged**